### DISCUSSION TITLE
Integration with Bandcamp

### DISCUSSION THREAD

**Discussion by prisse89** [Marekkon5/onetagger#159] [Ideas]
Hello!
Another great feature would be to integrate 1T with Bandcamp. Any idea if it is possible to do that?

**Comment by Marekkon5** [comment]
Hello, Bandcamp has only very few metadata available (title, artist, album, cover and genre). What tags do you expect from this? Thank you

**Reply by prisse89** [reply]
Hi,
These few metadata available in there (title, artist, album, cover, and genre) would be more than enough. Thank you also.
Cheers

**Comment by Marekkon5** [comment]
Added in latest commit, you can get the build from the Actions tab. 


BTW: next time please make an issue for feature requests / bugs, because I tend to forget this exists.
